website_sale_loyalty_suggestion_wizard_multi_gift/templates/wizard_templates.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- Wizard layout -->
+    <template
+        id="promotion_wizard_reward_card"
+        inherit_id="website_sale_loyalty_suggestion_wizard.promotion_wizard_reward_card"
+    >
+        <xpath
+            expr="//t[@t-if=&quot;reward.reward_type == 'product'&quot;]"
+            position="after"
+        >
+            <t t-if="reward.reward_type == 'multi_gift'">
+                <div class="row">
+                    <t t-foreach="reward.loyalty_multi_gift_ids" t-as="multi_gift_line">
+                        <div
+                            t-if="len(reward.loyalty_multi_gift_ids) > 1"
+                            class="fw-bolder pt-2 pb-1"
+                        >Choose Reward:</div>
+                        <t
+                            t-foreach="multi_gift_line.reward_product_ids"
+                            t-as="product"
+                        >
+                            <t
+                                t-call="website_sale_loyalty_suggestion_wizard.promotion_wizard_reward_product"
+                            >
+                                <t
+                                    t-set="reward_qty"
+                                    t-value="multi_gift_line.reward_product_quantity"
+                                />
+                                <t t-set="product" t-value="product" />
+                                <t t-set="id" t-value="multi_gift_line.id" />
+                                <t
+                                    t-set="optional_product"
+                                    t-value="len(multi_gift_line.reward_product_ids) > 1"
+                                />
+                                <t
+                                    t-set="option_checked"
+                                    t-value="product == multi_gift_line.reward_product_ids[:1]"
+                                />
```

Comment:
Maybe in another module or in this one, but I think it could be useful to add the product name including attributes to differentiate between variants. What do you think?

![image](https://github.com/user-attachments/assets/add446b7-c0da-496e-87cc-c5879f574ea1)

Reply:
Done!

![image](https://github.com/user-attachments/assets/b3e57524-8667-4570-b5bb-3ab76799a0ef)